docs/content/modeling/user-groups.mdx

```diff
@@ -21,11 +21,11 @@ import {
 
 <DocumentationNotice />
 
-In this guide you will learn how to add users to groups and grant groups access to an <ProductConcept section="what-is-an-object" linkName="object" /> using <ProductName format={ProductNameFormat.ProductLink}/>.
+This guide explains how to add users to groups and grant groups access to an <ProductConcept section="what-is-an-object" linkName="object" /> using <ProductName format={ProductNameFormat.ProductLink}/>.
 
 <CardBox title="When to use" appearance="filled">
 
-Adding a relationship tuple specifying that a group has a relation to an object is helpful in cases where you want to encompass a set of users with the same relation to an object. For example:
+Relationship tuples can specify that an entire group has a relation to an object, which is helpful when you want to encompass a set of users with the same relation to an object. For example:
 
 - Grant a group of `engineers` `viewer` access to `roadmap.doc`
```

Comment:
Nit: Reordering a bit so that we don't have back-to-back code callouts 
```suggestion
- Grant `viewer` access to a group a group of `engineers` in `roadmap.doc`
```